internal/serverinstall/serverinstall.go

```diff
@@ -0,0 +1,22 @@
+package serverinstall
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clicontext"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/internal/server/gen"
+)
+
+type Installer interface {
```

Comment:
Doc comment and also recommend commenting the interface methods. Take a look at some of our other interfaces we _usually_ do this.

Reply:
Ah right, of course!